ui-components/__tests__/navigation/Tabs.test.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+import { render, cleanup, fireEvent } from "@testing-library/react"
+import { Tabs, TabList, Tab, TabPanel } from "../../src/navigation/Tabs"
+
+afterEach(cleanup)
+
+describe("<Tabs>", () => {
+  it("displays the right CSS", () => {
```

Comment:
This is just testing implementation details - is there a way we can test component or prop functionality without looking at class names?

Reply:
I tried to at least test the left/right arrow keys like on TabNav but was unsuccessful. Spent a while trying to get the events to fire but something was off. Basically we'd be testing a third-party component anyway, so not sure how much we need to dig into that. Maybe write a test to simulate clicking on the different tabs and double-check if that works?

Reply:
Fair point on the 3rd party component :) I'll take a quick peek

Reply:
Looks like it's really well tested on their end. I took a peek just to see [how they did it](https://github.com/reactjs/react-tabs/blob/main/src/components/__tests__/Tabs-test.js) bc I also couldn't get the tabs to change, and it looks like they're pulling in another RTL sub-package called userEvent (🤷‍♀️) I would still recommend we change the first test though. I don't think we should rely on any of their class names, as we could pull a new version of this and it would break the test. Maybe we can just test our own prop? Something like:
` it("can pass a custom class name", () => {
    const { container } = render(
      <Tabs defaultIndex={1}>
        <TabList>
          <Tab className="other-tab">Other</Tab>
          <Tab>Default</Tab>
        </TabList>
        <TabPanel className="other-panel">OtherPanel</TabPanel>
        <TabPanel>DefaultPanel</TabPanel>
      </Tabs>
    )
    expect(container.getElementsByClassName("other-tab").length).toBe(1)
    expect(container.getElementsByClassName("other-panel").length).toBe(1)
  })`

Reply:
Thanks @emilyjablonski, I'll try that…good call on not hardcoding their internal class names. I'll see if I can remove one from our stylesheet in lieu of a custom one as well.